### PR TITLE
Add rosdep rules for git-lfs package

### PR DESCRIPTION
**Pull request by 708yamaguchi**
## Package name:

git-lfs

## Package Upstream Source:

https://github.com/git-lfs/git-lfs

## Purpose of using this:

I would like to download the trained data with git-lfs when using ROS packages that use DNN.

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - https://packages.debian.org/sid/git-lfs
- Ubuntu: https://packages.ubuntu.com/
   - https://packages.ubuntu.com/bionic/vcs/git-lfs
- Fedora: https://src.fedoraproject.org/browse/projects/
  - https://src.fedoraproject.org/rpms/git-lfs
- Arch: https://www.archlinux.org/packages/
  - https://archlinux.org/packages/community/x86_64/git-lfs/
- Gentoo: https://packages.gentoo.org/
  - https://packages.gentoo.org/packages/dev-vcs/git-lfs
- macOS: https://formulae.brew.sh/
  - N/A
- Alpine: https://pkgs.alpinelinux.org/packages
  - https://pkgs.alpinelinux.org/package/v3.7/community/x86/git-lfs
- NixOS/nixpkgs: https://search.nixos.org/packages
  - https://hydra.nixos.org/job/nixos/release-21.05/nixpkgs.git-lfs.x86_64-linux
- FreeBSD
  - https://freebsd.pkgs.org/13/freebsd-amd64/git-lfs-2.13.3.txz.html
- RHEL
  - https://koji.fedoraproject.org/koji/rpminfo?rpmID=21513967
- MacPorts
  - https://ports.macports.org/port/git-lfs/summary
- OpenSUSE
  - https://software.opensuse.org/package/git-lfs

### CODE REVIEW

File: rosdep/base.yaml
Comment (cottsay) [inline rosdep/base.yaml:1342]: CI is reporting that this package is not available for `stretch`, which looks to be correct.

```suggestion
  debian:
    '*': [git-lfs]
    stretch: null
```

Comment (708yamaguchi) [reply rosdep/base.yaml:1342]: Thank you so much!
This is exactly the writing style I was looking for.

**Comment by 708yamaguchi** [comment]
CI passed!

**Review by wjwwood — APPROVED** [review]
Waiting for second approval.

**Review by tfoote — APPROVED** [review]

**Pull request merged**
